src/ess/isissans/general.py

```diff
@@ -132,12 +148,43 @@ def get_calibrated_isis_detector(
 
 
 def get_monitor_data(
-    dg: LoadedFileContents[RunType], nexus_name: NeXusMonitorName[MonitorType]
+    dg: LoadedFileContents[RunType],
+    nexus_name: NeXusMonitorName[MonitorType],
+    spectrum_number: MonitorSpectrumNumber[MonitorType],
 ) -> NeXusComponent[MonitorType, RunType]:
+    """
+    Extract monitor data that was loaded together with detector data.
+
+    If the raw files is histogram data, Mantid stores this as a Workspace2D, where some
+    or all spectra corresponds to monitors.
+
+    Parameters
+    ----------
+    dg:
+        Data loaded with Mantid and converted to Scipp.
+    nexus_name:
+        Name of the monitor in the NeXus file, e.g. 'incident_monitor' or
+        'transmission_monitor'. USed when raw data is an EventWorkspace, where
+        monitors are stored in a group with this name.
+    spectrum_number:
+        Spectrum number of the monitor in the NeXus file, e.g. 0 for incident monitor
+        or 1 for transmission monitor. This is used when the raw data is a
```

Comment:
Are those example numbers correct? The default params are -1. Which also seems odd because both monitors have the same.